Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -576,17 +572,9 @@ class BrowserViewController: UIViewController {
         self.present(alert, animated: true, completion: nil)
     }
 
-    fileprivate func shouldRestoreTabs() -> Bool {
+    fileprivate func canRestoreTabs() -> Bool {
         guard let tabsToRestore = TabManager.tabsToRestore() else { return false }
```

Comment:
If there is a crash in `TabManger.tabsToRestore()` which is the method that does the NSUnarchiving wouldn't that cause the app to crash even before we display the `restoreTabsAlert`?